script/integration_test

```diff
@@ -1,34 +1,62 @@
-#!/bin/bash
-
-ROOT=$PWD
-export PATH=$ROOT:$PATH
+#!/usr/bin/env bash
+set -euo pipefail
+
+title() {
+    echo -e "\n\n#--------------------------------------------"
+    echo -e "# $@"
+    echo -e "#"
+}
+
+setup_environment() {
+    echo " -> Build the project."
+    go build
+
+    echo " -> Adding the current dir to PATH."
+    ROOT=$PWD
+    export PATH="$ROOT:$PATH"
```

Comment:
don't we have to clean up afterward by saving previous value by example or removing `$ROOT` value from `PATH`?

Reply:
This script is executed (-> subprocess). Not sourced (script run in your initial shell).

This won't alter the PATH variable of your shell.